Scripts/NeedDisabledSVE.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python3
+import os
+import subprocess
+import re
+import sys
+import tempfile
+import platform
+
+def ListContainsRequired(Features, RequiredFeatures):
```

Comment:
What's the benefit over this function over just using `in` directly?

Reply:
Removed and used directly.